packages/cli/lib/lib/webpack/prerender.js

```diff
@@ -24,9 +24,12 @@ module.exports = function(env, params) {
 			);
 			return '';
 		}
+		const { cwd } = env;
 
-		let preact = require('preact'),
-			renderToString = require('preact-render-to-string');
+		const preact = require(require.resolve(`${cwd}/node_modules/preact`));
```

Comment:
Currently, using a monorepo manager like lerna this workaround breaks the build process and returns:

`✖ ERROR Template execution failed: Error: Cannot find module './node_modules/preact'`

The `cwd` is located inside `root/packages/mypack` and is looking for a `root/packages/mypack/node_modules`, which however is located in `root/node_modules`. Node won't resolve correctly thanks to lerna and thus this will break.

I reverted the changes and logged destination. A revert worked but I think changing the destination to: `${__dirname}/../../../../preact` and `${__dirname}/../../../../preact-render-to-string` would work too.

Reply:
is there any workaround for this? I just moved a preact app into my monorepo to not have to maintain it in a separate repo, and I'm running into this exact error.

Reply:
With Lerna and Yarn workspaces no. If you disable yarn workspaces this should work with Lerna though.

Reply:
Fixed in https://github.com/preactjs/preact-cli/pull/1418